libcxxabi/src/private_typeinfo.cpp

```diff
@@ -51,6 +51,21 @@
 #include <atomic>
 #endif
 
+#if __has_feature(ptrauth_calls)
+#include <ptrauth.h>
+#endif
+
+
+template<typename T>
+static inline
+T *
+get_vtable(T *vtable) {
+#if __has_feature(ptrauth_calls)
```

Comment:
Could you please give this a more descriptive name than `get_vtable`, because we already have the vtable in most cases when we call this so it's a touch confusing.

Maybe a name that ties it to `ptrauth`?

Reply:
Hmmm, it may be possible for __builtin_get_vtable() to be used as instead, but I'm not sure what the considerations were when this code was written (or if it predates that builtin?)

Reply:
The dynamic_cast machinery had to strip but `__builtin_get_vtable` always authenticates, so as-is this probably needs to be a manual `ptrauth_strip` yeah